src/jit/gentree.cpp

```diff
@@ -1511,6 +1511,24 @@ bool GenTree::Compare(GenTree* op1, GenTree* op2, bool swapOK)
                     Compare(op1->gtArrOffs.gtIndex, op2->gtArrOffs.gtIndex) &&
                     Compare(op1->gtArrOffs.gtArrObj, op2->gtArrOffs.gtArrObj));
 
+        case GT_PHI:
+        {
+            GenTreePhi::UseIterator op1Use    = op1->AsPhi()->Uses().begin();
```

Comment:
Might be useful to also mention that the order matters for equality, even though it's otherwise irrelevant. That's a bit peculiar but I don't think this code is ever used.